All EIPIP Meetings/Meeting 007.md

```diff
@@ -0,0 +1,527 @@
+The SEO implications were actually in favor of moving it, because we would get more traffic by having it be ethereum.org/eips rather than what it is now.
+
+**Tim Beiko**
+Got it.
+
+**Artem Vorotnikov**
+Also, I suppose we could integrate EIPs into the unified translation platform. I don'tr know if this has been considered.
+
+**Edson Ayllon**
+Yeah, that was another argument for, that they could help with translations.
+
+**Hudson Jameson**
+We could do the translations without the integration, though. It's not like fitting like a glove when you do that. I'm not super versed in the way they're doing translations right now. Also, that would be a lot. I don't think they would want to take that on for translating on the go every EIP that comes in. That is something we should strive for, for sure. But I don't know if they're gonna want to take that on. That would be a question for them.
+
+**James Hancock**
+Even just translating EIP 1 would be really hard. We don't even know what it all means.
+
+**Hudson Jameson**
+And the wording is overly technical, Not overly technical for what it is, but overly technical for a translator. How would you translate EFI?
+
+**Pooja Ranjan**
+Recently I discovered a script translator on Google. I'm not sure if that would be helpful for EIP 1. Somebody can try that.
+
+**Alita**
+I would say, in general, you should be very careful with anything automated, because automatic translations are typically like the Holy Grail. So we're not there yet, if that makes sense.
+
+**Hudson Jameson**
+You're saying it's a rough translation when that happens.
+
+**Alita**
+Yeah, basically, the ability to translate effectively between languages is the goal of current research in speech recognition stuff. So I find it highly unlikely you would find one actually usable. You'd probably be better off finding a good translator.
+
+**Hudson Jameson**
+Yeah, or a combination. Do people do combinations? Anybody heard of that? They do the auto translate and then have someone come in and clean it up?
+
+**James Hancock**
+My experience with Japanese is that it's really, really doesn't work. But that's probably because Japanese is really, really different.
+
+Overall, I would say if I was looking for a translator for EIP 1, I would probably want basically a lawyer-like level of understanding of language while translating. Because a lot of weight is given to a lot of these words, and not knowing nuance or context to certain things, especially technical ones.
+
+I know that to be a patent attorney, you have to have a dual degree of law and of engineering, specifically because of this kind of stuff. I feel like this is a little bit of distraction, but crowd sourcing EIP 1 translations - I wouldn't be really comfortable.
+
+**Pooja Ranjan**
+So there is this team of translation led by [unclear - Deon?] from the Ethereum Foundation. And people find it's worth having the translation of EIP 1. We can probably reach out to her and try to get it.
```

Comment:
[unclear - Deon?] - Taeyeon

Reply:
And if people find it's worth having the translation of EIP 1, we can probably reach out to her and try to get it.